attribution-data-file-share/src/test/java/gov/cms/ab2d/attributionDataShare/AttributionDataShareHandlerTest.java

```diff
@@ -0,0 +1,29 @@
+package gov.cms.ab2d.attributionDataShare;
+
+import gov.cms.ab2d.testutils.AB2DPostgresqlContainer;
+import gov.cms.ab2d.testutils.TestContext;
+
+import org.junit.jupiter.api.Test;
+import org.testcontainers.containers.PostgreSQLContainer;
+import org.testcontainers.junit.jupiter.Container;
+import org.testcontainers.junit.jupiter.Testcontainers;
+
+import static org.junit.jupiter.api.Assertions.assertDoesNotThrow;
+
+@Testcontainers
+public class AttributionDataShareHandlerTest {
+
+    @SuppressWarnings({"rawtypes", "unused"})
+    @Container
+    private static final PostgreSQLContainer POSTGRE_SQL_CONTAINER = new AB2DPostgresqlContainer();
+
+    @Test
+    void attributionDataShareInvoke() {
+        AttributionDataShareHandler attributionDataShareHandler = new AttributionDataShareHandler();
+
+        assertDoesNotThrow(() -> {
+            attributionDataShareHandler.handleRequest(null, System.out, new TestContext());
+        });
+    }
+
```

Comment:
I need to look into Junit more, but I am wondering what other tests we might want here? Maybe we could verify that a file was created on the disk? Or is that out of scope of the unit test?